webapp/src/app/shared/global-events.service.ts

```diff
@@ -17,6 +17,8 @@ export class GlobalEventsService {
 	offsetMap = new Subject<Point>();
 	toggleVisibility = new Subject<void>();
 	showAddEntityMenu = new Subject<Point>();
+
+  updateCoords = new Subject<Point3 | null>();
```

Comment:
use `undefined` instead of `null`, also wrong indentation
```suggestion
	updateCoords = new Subject<Point3 | undefined>();
```